imageengine/class-imagecdn.php

```diff
@@ -34,7 +34,8 @@ public function __construct() {
 			add_filter( 'the_content', array( self::class, 'rewrite_the_content' ), 100 );
 
 			// Resource hints.
-			add_action( 'wp_head', array( self::class, 'add_head_tags' ), 0 );
+			//add_action( 'wp_head', array( self::class, 'add_head_tags' ), 0 );
+      add_action( 'send_headers', array( self::class, 'add_headers' ), 0 );
```

Comment:
I'll need to remember how to run the code-style-fixer that corrects indentation problems like this (the WordPress style is horrible, but I stuck to it to avoid code review problems).  In this case, the spacing should be tabs, not spaces.